nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/uidgen/KeyValueStoreUIDSequencer.java

```diff
@@ -50,10 +48,7 @@
 
     @Override
     public void init() {
-        storeName = Framework.getService(ConfigurationService.class).getProperty(STORE_NAME_PROPERTY);
-        if (isBlank(storeName)) {
-            storeName = DEFAULT_STORE_NAME;
-        }
+        storeName = Framework.getService(ConfigurationService.class).getString(STORE_NAME_PROPERTY, DEFAULT_STORE_NAME);
```

Comment:
This doesn't take into account blank values though (without any ERROR log), does it? It's important to keep this behavior.

Reply:
It's now handle in `ConfigurationService`.